core/action.class.inc.php

```diff
@@ -167,6 +168,46 @@ protected function DoCheckIfHasTrigger()
 			$this->m_aCheckWarnings[] = Dict::S('Action:WarningNoTriggerLinked');
 		}
 	}
+
+	/**
+	 * @since 3.2.0 N°5472 method creation
+	 */
+	public function DisplayBareRelations(WebPage $oPage, $bEditMode = false)
+	{
+		parent::DisplayBareRelations($oPage, false);
+
+		$this->GenerateLastExecutionsTab($oPage, $bEditMode);
+	}
+
+	/**
+	 * @since 3.2.0 N°5472 method creation
+	 */
+	protected function GenerateLastExecutionsTab(WebPage $oPage, $bEditMode)
+	{
+		if ($bEditMode) {
+			return;
+		}
+
+		$oPage->SetCurrentTab('action_errors', Dict::S('Action:last_executions_tab'), Dict::S('Action:last_executions_tab+'));
```

Comment:
```suggestion
		$oPage->AddAjaxTab('Action:last_executions_tab', XXX, true /* bCache */, Dict::S('Action:last_executions_tab'), AjaxTab::ENUM_TAB_PLACEHOLDER_LIST);
```

For the ajax tab you can try something like this

Reply:
TODO reported in description
Closing conversation